pkg/report/testdata/linux/report/743

```diff
@@ -0,0 +1,224 @@
+[  456.871366][    C1]  ? __folio_rmap_sanity_checks+0x120/0x700
+[  456.877308][    C1]  __folio_rmap_sanity_checks+0x120/0x700
+[  456.883046][    C1]  folio_remove_rmap_ptes+0x3b/0xaf0
+[  456.888367][    C1]  ? page_table_check_clear+0x187/0x700
+[  456.893919][    C1]  ? page_table_check_clear+0x4f3/0x700
+[  456.899501][    C1]  ? page_table_check_clear+0x187/0x700
+[  456.905089][    C1]  unmap_page_range+0x1e59/0x41c0
+[  456.910159][    C1]  ? __pfx_unmap_page_range+0x10/0x10
+[  456.915569][    C1]  ? mas_find+0x987/0xbc0
+[  456.919906][    C1]  ? unmap_vmas+0x144/0x580
+[  456.924420][    C1]  unmap_vmas+0x399/0x580
+[  456.928798][    C1]  ? __pfx_unmap_vmas+0x10/0x10
+[  456.933676][    C1]  exit_mmap+0x248/0xb50
+[  456.937950][    C1]  ? uprobe_clear_state+0x20f/0x290
+[  456.943161][    C1]  ? __pfx_exit_mmap+0x10/0x10
+[  456.947949][    C1]  ? __mutex_unlock_slowpath+0x1cd/0x700
+[  456.953604][    C1]  ? __pfx_exit_aio+0x10/0x10
+[  456.958321][    C1]  ? uprobe_clear_state+0x274/0x290
+[  456.963534][    C1]  __mmput+0x118/0x420
+[  456.967640][    C1]  exit_mm+0x1da/0x2c0
+[  456.971735][    C1]  ? __pfx_exit_mm+0x10/0x10
+[  456.976360][    C1]  ? rcu_is_watching+0x15/0xb0
+[  456.981142][    C1]  do_exit+0x648/0x22e0
+[  456.985330][    C1]  ? do_raw_spin_lock+0x121/0x290
+[  456.990364][    C1]  ? __pfx_do_exit+0x10/0x10
+[  456.994994][    C1]  do_group_exit+0x21c/0x2d0
+[  456.999602][    C1]  ? lockdep_hardirqs_on+0x9c/0x150
+[  457.004880][    C1]  get_signal+0x1286/0x1340
+[  457.009417][    C1]  arch_do_signal_or_restart+0x9a/0x750
+[  457.015006][    C1]  ? count_memcg_event_mm+0x21/0x260
+[  457.020330][    C1]  ? __pfx_arch_do_signal_or_restart+0x10/0x10
+[  457.026531][    C1]  ? exit_to_user_mode_loop+0x40/0x110
+[  457.032004][    C1]  exit_to_user_mode_loop+0x75/0x110
+[  457.037320][    C1]  do_syscall_64+0x2bd/0x3b0
+[  457.041925][    C1]  ? lockdep_hardirqs_on+0x9c/0x150
+[  457.047253][    C1]  ? entry_SYSCALL_64_after_hwframe+0x77/0x7f
+[  457.053325][    C1]  ? clear_bhb_loop+0x60/0xb0
+[  457.058030][    C1]  entry_SYSCALL_64_after_hwframe+0x77/0x7f
+[  457.063927][    C1] RIP: 0033:0x7f7ace38e929
+[  457.068361][    C1] Code: Unable to access opcode bytes at 0x7f7ace38e8ff.
+[  457.075399][    C1] RSP: 002b:00007f7acc1f6038 EFLAGS: 00000246 ORIG_RAX: 000000000000012b
+[  457.083837][    C1] RAX: 0000000000010106 RBX: 00007f7ace5b6080 RCX: 00007f7ace38e929
+[  457.091854][    C1] RDX: 0000000000010106 RSI: 00002000000000c0 RDI: 0000000000000003
+[  457.099854][    C1] RBP: 00007f7ace410b39 R08: 0000000000000000 R09: 0000000000000000
+[  457.107865][    C1] R10: 0000000000000002 R11: 0000000000000246 R12: 0000000000000000
+[  457.115864][    C1] R13: 0000000000000001 R14: 00007f7ace5b6080 R15: 00007ffdb04f0738
+[  457.123853][    C1]  </TASK>
+[  457.126909][    C1] Kernel panic - not syncing: kernel: panic_on_warn set ...
```

Comment:
FWIW I think it should be fine to stop at this point and not keep the whole huge log.

Reply:
These are just input data, so probably not that critical, but I'll keep in mind that these may indeed be shorter.